librz/core/cmd/cmd_print.c

```diff
@@ -5339,89 +5339,67 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
-			} else {
-				char *str = NULL;
-				bool big_endian = rz_config_get_b(core->config, "cfg.bigendian");
-				switch (input[1]) {
-				case '*': // "pc*" // rizin commands
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_RIZIN);
-					break;
-				case 'a': // "pca" // GAS asm
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_ASM);
-					break;
-				case 'b': // "pcb" // bash shellscript
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_BASH);
-					break;
-				case 'n': // "pcn" // nodejs
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_NODEJS);
-					break;
-				case 'g': // "pcg" // golang
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_GOLANG);
-					break;
-				case 'k': // "pck" kotlin
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_KOTLIN);
-					break;
-				case 's': // "pcs" // swift
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_SWIFT);
-					break;
-				case 'r': // "pcr" // Rust
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_RUST);
-					break;
-				case 'o': // "pco" // Objective-C
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_OBJECTIVE_C);
-					break;
-				case 'J': // "pcJ" // java
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_JAVA);
-					break;
-				case 'y': // "pcy" // yara
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_YARA);
-					break;
-				case 'j': // "pcj" // json
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_JSON);
-					break;
-				case 'p': // "pcp" // python
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_PYTHON);
-					break;
-				case 'h': // "pch" // C half words with asm
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, big_endian ? RZ_LANG_BYTE_ARRAY_C_CPP_HALFWORDS_BE : RZ_LANG_BYTE_ARRAY_C_CPP_HALFWORDS_LE);
-					break;
-				case 'w': // "pcw" // C words with asm
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, big_endian ? RZ_LANG_BYTE_ARRAY_C_CPP_WORDS_BE : RZ_LANG_BYTE_ARRAY_C_CPP_WORDS_LE);
-					break;
-				case 'd': // "pcd" // C double-words with asm
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, big_endian ? RZ_LANG_BYTE_ARRAY_C_CPP_DOUBLEWORDS_BE : RZ_LANG_BYTE_ARRAY_C_CPP_DOUBLEWORDS_LE);
-					break;
-				default: // "pc" // C bytes
-					str = rz_lang_byte_array(core->block, len < 0 ? 0 : len, RZ_LANG_BYTE_ARRAY_C_CPP_BYTES);
-					break;
-				}
-				if (str) {
-					rz_cons_println(str);
-					free(str);
-				}
+			char *str = NULL;
+			bool big_endian = rz_config_get_b(core->config, "cfg.bigendian");
+			switch (input[1]) {
+			case 'A': // "pcA" .bytes with instructions in comments
```

Comment:
@wargio I see in https://github.com/rizinorg/rizin/pull/2444/files# you converted these commands to rzshell, but you also added this block of code. Is there any reason for that considering that it is for oldshell code?

@imbillow I think we should add this command to rzshell rather than here.

Reply:
I actually found some inconsistencies between rzshell and the old `pc` command

Reply:
because before i thought we wanted to keep the old commands and remove them only at job done but then we changed idea, so yeah, these oldshell stuff can be removed.

Reply:
I think the rzshell should be considered the "right" one at this point, unless there's something really wrong. Please remove that oldshell code since you are at it.